src/test/java/seedu/weme/model/ModelManagerTest.java

```diff
@@ -116,11 +113,6 @@ public void equals() {
         // different memeBook -> returns false
         assertFalse(modelManager.equals(new ModelManager(differentMemeBook, userPrefs)));
 
-        // different filteredList -> returns false
```

Comment:
can keep this test, just use a TagContainsKeywordsPredicate

Reply:
Will look into it. Might need to change the logic of TagContainsKeywordsPredicate again.